GSoC26_H/inference/normalize_triples.py

```
@@ -0,0 +1,404 @@
+                print(f"    Rate limited — waiting {wait}s...")
+                time.sleep(wait)
+                continue
+            if resp.status_code != 200:
+                raise ValueError(f"HTTP {resp.status_code}")
+            choice = resp.json()["choices"][0]["message"]
+            raw = (choice.get("content") or
+                   choice.get("reasoning_content") or "").strip()
+            if not raw:
+                raise ValueError("Empty response")
+            return parse_llm_answer(raw, valid_uris)
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)